lib/crawlers/company_crawler/company_crawler.rb

```diff
@@ -0,0 +1,80 @@
+require_relative "../../priority_spidr/priority_spidr"
+require_relative "../base_crawler"
+require "cgi"
+require "pathname"
+require "csv"
+require "uri"
+
+module Crawlers
+  class CompanyCrawler < BaseCrawler
+    ROOT = Pathname.new(__FILE__).parent.parent.parent.parent
+    # Requires a url string
+    #
+    # @param url [String]
+    #
+    # @param stubs_path [Pathname, NilClass]
+    def initialize(url, stubs_path = Pathname.new(__FILE__).parent + "careers_page_stubs.txt") # rubocop:disable Style/StringConcatenation
+      super(url, stubs_path)
+      @url_regex = Regexp.new('https?://(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b(?:[-a-zA-Z0-9@:%_\+.~#?&//=]*)')
+      @ats_stubs = load_ats_stubs
+    end
+
+    private
+
+    # Load and return stubs list from `storage/csv/ats_systems.csv`
+    #
+    # @return [Array<String>]
+    def load_ats_stubs
+      stubs = []
+      ats_csv = CSV.parse(File.read(ROOT + "storage" + "csv" + "ats_systems.csv"), headers: true) # rubocop:disable Style/StringConcatenation
```

Comment:
Should be able to write something like 'storage/csv/ats_systems.csv' here

Reply:
I was doing it like that with the `pathname` gem b/c it automatically uses the correct path separator based on the system, so on Unix it produces 'storage/csv/ats_systems.csv', but on windows it produces 'storage\csv\ats_systems.csv'. 
If we want to assume this is guaranteed to only run on a Unix system (seems to currently be the case), then I can ditch the `pathname` gem.